clients/imodels-client-authoring/package.json

```diff
@@ -34,12 +34,13 @@
   "dependencies": {
     "@azure/storage-blob": "^12.7.0",
     "@itwin/imodels-client-management": "workspace:*",
-    "@itwin/object-storage-azure": "^2.0.0",
-    "@itwin/object-storage-core": "^2.0.0"
+    "@itwin/object-storage-azure": "^2.2.5",
+    "@itwin/object-storage-core": "^2.2.5"
   },
   "devDependencies": {
     "@itwin/eslint-plugin": "~3.7.8",
     "@itwin/imodels-client-common-config": "workspace:*",
+    "@types/node": "^22.3.0",
```

Comment:
prob want to use v20 not 22 since 22 isnt lts yet

Reply:
thanks, fixed